### ISSUE TITLE
g131 test failure

### ISSUE THREAD

**Issue by joebo** [jsoftware/j-playground#35]
The test case g131 has several failures. The first failure is below. The full test can be run from the playground link

[Playground Link](https://jsoftware.github.io/j-playground/bin/html2/#url=https://raw.githubusercontent.com/jsoftware/j-playground/j903_wasm/test/g131.ijs)

Note: It may take a minute or so to run in the playground. I'm also not clear how to diagnose this

```
  1:@:(dbr bind Debug)@:(9!:19)2^_44[(prolog [ echo^:ECHOFILENAME) './g131.ijs'
1
   NB. %. ------------------------------------------------------------------
   
   NB. mdiv failed on small ct
   ct   =: 9!:18''
   9!:19[2^_40

   
   X    =: +/ . *
   en   =: 1&{@(,&1 1)@$
   norm =: (%:@X +)@,
   
   NB. given n, find m, the recursion split value
   splitm =: 3 : 0
rawm =. <. -: y
xferamt =. (7 (17 b.) y) { 0 0 1 2 2 2 1 0
if. y>rawm+xferamt do. rawm =. rawm+xferamt end.
rawm
)
   
   qr =: 3 : 0
 n  =. en y
 if. 1>:n do.
  (((* %) ;&,. ,~@en@[ $ ]) norm) y
 else.
  m  =. splitm n
  a0 =. m{."1 y
  a1 =. m}."1 y
  t0 =. qr a0
  q0 =. >@{. t0
  r0 =. >@{: t0
  c  =. (+|:q0) X a1
  t1 =. qr a1 - q0 X c
  q1 =. >@{. t1
  r1 =. >@{: t1
  (q0,.q1);(r0,.c),(-n){."1 r1
 end.
)
   
   em   =:splitm@#                    NB.  m  =. >.-:#y.
   mm   =: ,~@em                      NB.  mm =: m,m
   ai   =: rinv@(mm {. ])             NB.  ai =: rinv mm{.y.
   di   =: rinv@(mm }. ])             NB.  di =: rinv mm}.y.
   bee  =: (em , em - #) {. ]         NB.  b  =: (m,m-n){.y.
   bx   =: -@(ai X bee X di)          NB.  bx =: - ai X b X di
   r4   =: (ai,.bx) , (-@# {."1 di)   NB.  (ai,.bx),(-n){."1 di
   rinv =: r4`% @. (1&>:@#)
   
   minv =: (|.@$ ($,) (rinv@] X +@|:@[)&>/@qr) " 2
   mdiv =: (%.@] +/ . * [) " _ 2
   
   id=: =&i.&#
   
   4 19 -: $%.?19 4$2
1
   6 10 -: $%.?10 6$100
1
   7 11 -: $%.?11 7$100
1
   5 7  -: $%.j./?2 7 5$100
1
   
   1       -: %.1
1
   (%4)    -: %.4
1
   (%3.5)  -: %.3.5
1
   (%4j_5) -: %.4j_5
1
   
   (%. -: minv) 1
1
   (%. -: minv) 1+?12369        NB. minv failed on 0
1
   (%. -: minv) (+ 0&=)o.?1721
1
   (%. -: minv) j./1+?100 2001
1
   
   (1e_1 > >./ | , b-minv a) *. 1e_1>>./|,(id a)-a X b=:%.a=:_50000+?10 10$10000
|domain error: RUN1
|   (0.1>>./|,b-minv a)*.0.1>>./|,(id a)-a X b=:    %.a=:_50000+?10 10$10000
|[-67] //test/g131.ijs
```

**Comment by joebo** [comment]
This exact error also shows up when compiling the linux version with tests. make -f makefile.linux clean j , now that support for running tests under the linux_wasm build has been added... This means we can debug the C code and not the emitted WASM.

**Comment by joebo** [comment]
fixed with 517e00a2f9c77a2cbe65bd89d624331fb90fe563
